packages/paste-website/src/pages/components/disclosure/index.mdx

```diff
@@ -0,0 +1,499 @@
+  <DisclosureHeading as="h2" variant="heading60">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+### Contained Disclosure
+
+The `contained` variant of the Disclosure takes the default composition and wraps it in the [Paste Card](/components/card) component. In this instance we do not expose the props used by Card, but all previously mentioned Disclosure props behave in the same way as the default variant.
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure variant="contained">
+  <DisclosureHeading as="h2" variant="heading10">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure variant="contained">
+  <DisclosureHeading as="h2" variant="heading20">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure variant="contained">
+  <DisclosureHeading as="h2" variant="heading30">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure variant="contained">
+  <DisclosureHeading as="h2" variant="heading40">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure variant="contained">
+  <DisclosureHeading as="h2" variant="heading50">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure variant="contained">
+  <DisclosureHeading as="h2" variant="heading60">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+### Controlling initial state
+
+You can control the intial state of the disclosure to be expanded or collapsed by setting the `visible` prop.
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure visible>
+  <DisclosureHeading as="h2" variant="heading30">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+## States
+
+### Disabled
```

Comment:
Should we have a disabled pointer for this one?

Reply:
Frustratingly because the primitive adds an inline style for `pointer-events: none` it removes the ability to change any pointer based styles for that element.